allennlp/commands/train.py

```diff
@@ -191,6 +205,9 @@ def train_model(params: Params,
         For caching data pre-processing.  See :func:`allennlp.training.util.datasets_from_params`.
     cache_prefix : ``str``, optional
         For caching data pre-processing.  See :func:`allennlp.training.util.datasets_from_params`.
+    cache_method : ``str``, optional (default='jsonpickle')
+        Data caching method. By default use `jsonpickle`, which is slow but portable. Alternatively,
+        `pickle` can be much faster but may not be portable across different Python versions.
```

Comment:
You should also note here that `pickle` is incompatible with lazy dataset readers.  I tried this originally, but it doesn't work.